core/src/main/java/org/frankframework/pipes/CsvParserPipe.java

```diff
@@ -117,10 +115,10 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 
 	private void processCsvRecord(final CSVRecord csvRecord, final SaxDocumentBuilder document) throws PipeRunException {
 		try (SaxElementBuilder element = document.startElement("record")) {
-			for(Entry<String,String> entry: csvRecord.toMap().entrySet()) {
-				String key = XmlEncodingUtils.stripNonValidXmlCharacters(entry.getKey().replace(' ', '_'), false);
-				if(getHeaderCase() != null) {
-					key = getHeaderCase()==HeaderCase.LOWERCASE ? key.toLowerCase() : key.toUpperCase();
+			for (Entry<String,String> entry: csvRecord.toMap().entrySet()) {
+				String key = entry.getKey();
```

Comment:
Knutsel is eruit en de 'cleanse' methode dekt ook spaties, unit test voor toegevoegd om dit te bewijzen